src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -8,6 +8,19 @@ namespace Microsoft.CodeAnalysis
 {
     internal partial class ConstantValue
     {
+        /// <summary>
```

Comment:
It is probably worth noting that we choose the "cannonical" NaN with the sign bit set. Both positive and negative NaN, with no diagnostic payload are considered "cannonical".